app/controllers/campaigns_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CampaignsController < ApplicationController
+  before_action :find_campaign, only: %i[show edit update destroy]
+
+  def index
+    @campaigns = Campaign.all
+  end
+
+  def show; end
+
+  def new
+    @campaign = Campaign.new
+  end
+
+  def edit; end
+
+  def create
+    @campaign = current_user.campaigns.new(campaigns_params)
+    if @campaign.save
+      flash[:notice] = t('.successfully_create')
+      redirect_to action: 'index'
+    else
+      flash[:alert] = t('.didn_t_create')
+      render 'new'
+    end
+  end
+
+  def update
+    if @campaign.update(campaigns_params)
+      flash[:notice] = t('.successfully_update')
+      redirect_to action: 'index'
+    else
+      flash[:alert] = t('.didn_t_update')
+      render 'edit'
+    end
+  end
+
+  def destroy
+    if @campaign.destroy
+      flash[:notice] = t('.successfully_deleted')
+      redirect_to action: 'index'
+    else
+      flash[:alert] = t('.didn_t_delete')
+    end
+  end
+
+  private
+
+  def find_campaign
```

Comment:
let's rename it to load_campaign and share it with everyone to be consistent in the application.